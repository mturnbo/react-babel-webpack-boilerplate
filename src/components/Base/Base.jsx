import React from 'react';
import './base.css';

const Base = ({title, content}) => {
  return (
    <div className="base">
      <div className="base-header">{title}</div>
      <div className="base-content">{content}</div>
    </div>
  );
};

export default Base;
