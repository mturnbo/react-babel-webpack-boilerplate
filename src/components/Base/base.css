.base {
  color: darkblue;
  font-size: 12px;
}
