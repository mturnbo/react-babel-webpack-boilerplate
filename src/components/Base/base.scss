.base {
  width:400px;
  -webkit-border-radius: 0px 20px 0px 20px;
  -moz-border-radius: 0px 20px 0px 20px;
  border-radius: 0px 20px 0px 20px;
  border:2px solid #333333;
  background-color:#84ADE3;
  -webkit-box-shadow: #B3B3B3 10px 10px 10px;
  -moz-box-shadow: #B3B3B3 10px 10px 10px;
  box-shadow: #B3B3B3 10px 10px 10px;
  margin: 20px;
}

.base-header{
  font-size: 20px;
  font-weight: bold;
  margin: 5px;
  color: #000000;
  border-bottom: 1px solid #000000;
}

.base-content {
  color: #333333;
  font-size: 12px;
  margin: 10px 10px 20px 10px;
}

.base-button {
  text-align: right;
}