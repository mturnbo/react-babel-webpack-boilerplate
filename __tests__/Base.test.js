import React from 'react';
import { mount } from 'enzyme';
import Base from '../src/components/Base';

const wrapper = mount(<Base title="Test" content="Blah blah blah" />);

test('Base Component Renders', () => {
  expect(wrapper).toMatchSnapshot();
});
