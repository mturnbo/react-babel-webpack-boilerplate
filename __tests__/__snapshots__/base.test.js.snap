// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base Component Renders 1`] = `
<Base
  content="Blah blah blah"
  title="Test"
>
  <div
    className="base"
  >
    <div
      className="base-header"
    >
      Test
    </div>
    <div
      className="base-content"
    >
      Blah blah blah
    </div>
  </div>
</Base>
`;
